buildlib/pyverbs_functions.cmake

```diff
@@ -9,6 +9,7 @@ function(rdma_cython_module PY_MODULE)
     include_directories(${PYTHON_INCLUDE_DIRS})
     add_custom_command(
       OUTPUT "${CFILE}"
+      MAIN_DEPENDENCY "${PYX}"
```

Comment:
What does this change? I mean, cython will fail if you don't provide pyx files anyway.

Reply:
I usually delete the build/ directory before I call ninja, so I wasn't aware.

Reply:
Don't do things like that, it hides defects like this that effect the rest of us.